### ISSUE TITLE
rasa_core model won't train when story starts with utterance

### ISSUE THREAD

**Issue by joseariel** [RasaHQ/rasa_core#300]
**Rasa Core version**:
0.7.9

**Python version**: 
Python 3.4.2 (default, Oct  8 2014, 10:45:20) 

**Operating system** (windows, osx, ...):
Debian Jessie

**Issue**:
The rasa_core model won't train when the stories.md file has stories that are initiated by the bot, as opposed to the end-user. For instance, we're trying to train a model to start by asking 'How can we be of service?' and based on the response from the end-user take different subsequent actions (utterances).

Please see the following:

```yaml
## forgot_my_password_978as6df8a7df
    - utter_how_can_I_help
* _forgot_my_password
    - utter_application_assist

```

**Content of domain file** (if used & relevant):
```yaml
intents:
 - hello
 - forgot_my_password
 - thankyou

templates:
  utter_greet:
    - Hello!
  utter_how_can_I_help:
    - How can we be of service?
  utter_application_assist:
   - I'll be happy to assist you. Please visit http://... to reset your password.

actions:
  - utter_greet
  - utter_how_can_I_help
  - utter_application_assist

```

**Comment by tmbo** [comment]
What is it that happens after you trained that, is there an error?

**Comment by joseariel** [comment]
Hi Tom, thank you for your prompt response.

It was breaking the training process for the Agent, although I believe this may have been caused by conflicting rasa_core versions. After a lot of trial and error without success, I decided to throw everything away and start from scratch with the latest version of rasa_core. It is now working properly. 

I have another question about accuracy, not sure if this is the best place to post it or if I should create a new thread? Can't get some of my NLU intents to increase their confidence past 40%. Just wondering if you have any suggestions or could point me in the right direction.

Appreciate your help.

Joseariel
